server/src/test/java/io/crate/integrationtests/PrivilegesIntegrationTest.java

```diff
@@ -145,20 +147,21 @@ public void testGrantPrivilegeToSuperuserThrowsException() {
     @Test
     public void testApplyPrivilegesToUnknownUserThrowsException() {
         Asserts.assertSQLError(() -> executeAsSuperuser("grant DQL to unknown_user"))
-            .hasPGError(INTERNAL_ERROR)
+            .hasPGError(UNDEFINED_OBJECT)
             .hasHTTPError(NOT_FOUND, 40410)
             .hasMessageContaining("Role 'unknown_user' does not exist");
     }
 
     @Test
     public void testApplyPrivilegesToMultipleUnknownUsersThrowsException() {
         Asserts.assertSQLError(() -> executeAsSuperuser("grant DQL to unknown_user, also_unknown"))
-            .hasPGError(INTERNAL_ERROR)
+            .hasPGError(UNDEFINED_OBJECT)
             .hasHTTPError(NOT_FOUND, 40410)
             .hasMessageContaining("Roles 'unknown_user, also_unknown' do not exist");
     }
 
     @Test
+    @UseRandomizedSchema(random = false)
```

Comment:
Why is it necessary to disable schema randomization for all of these?

Reply:
Couldn't we ensure these other sessions also inherit the randomized schema?

The problem is that with `doc`, the index name matches the table name, but with custom schema's the encoding scheme is more complex. So there is a lot of logic that could happen to work for `doc`, but would fail with a custom schema so I'm a bit reluctant to just disable the sanity checks like this

Reply:
>Couldn't we ensure these other sessions also inherit the randomized schema?

I cannot use sessions without creating a user anymore. Do you mean we should keep using random schemas and adjust table access a bit? 
Smth like `sqlExecutor..getSessionSettings().searchPath().currentSchema() + tableName` instead of `tableName`

> So there is a lot of logic that could happen to work for doc, but would fail with a custom schema

But that's what we have on master, this PR only aligns with it. On master all tables are created under "doc" schema in all tests using `createSuperUserSession` or `createUserSession` (unless `search_path` modified explicitly like in `testGrantWithCustomDefaultSchema`).

Reply:
In other words, current state of master is actually confusing: we do have enabled randomization of schemas but schemas are always "doc" for tests using "session-without-a-real-user"